src/main/java/seedu/address/logic/parser/alias/AliasCommandParser.java

```diff
@@ -19,14 +23,15 @@ public AliasCommand parse(String args) throws ParseException {
         requireAllNonNull(args);
         ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args, PREFIX_ALIAS, PREFIX_COMMAND);
 
-        String aliasName = argMultimap.getValue(PREFIX_ALIAS).orElse("");
-        String command = argMultimap.getValue(PREFIX_COMMAND).orElse("");
-
-        if (aliasName.isEmpty() || command.isEmpty()) {
+        if (argMultimap.getValue(PREFIX_ALIAS).isEmpty() || argMultimap.getValue(PREFIX_COMMAND).isEmpty()) {
             throw new ParseException(
                     String.format(MESSAGE_INVALID_COMMAND_FORMAT, AliasCommand.MESSAGE_USAGE));
         }
 
+        String[] parsedArgs = args.split("\\s+", 3);
```

Comment:
Perhaps abstract the 3 into a variable to avoid magic numbers

Reply:
Good point!